tests/Sdk/Unit/Infrastructure/Workflow/WorkflowRunnerTest.php

```diff
@@ -5,33 +5,34 @@
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
-namespace Sdk\Unit\Infrastructure\Workflow;
+namespace SprykerSdk\Sdk\Unit\Infrastructure\Service;
```

Comment:
incorrect namespace. Must be `SprykerSdk\Sdk\Unit\Infrastructure\Unit`.

Reply:
I guess SprykerSdk\Sdk\Unit\Infrastructure\Workflow

Reply:
correct